cea/optimization/supportFn.py

```diff
@@ -64,9 +64,8 @@ def calcQmax(fName, filepath, gV):
     :return: Qmax: maximum heating power [W]
     :rtype: float
     """    
-    HeatfeatureList = ["T_sst_heat_return_netw_total", "T_sst_heat_supply_netw_total", "mdot_DH_netw_total"]
-    buildDemand = extractDemand(filepath+'//'+fName, HeatfeatureList, gV.DAYS_IN_YEAR)
-    Qmax = np.max(buildDemand)
+    Q_DHNf_W = pd.read_csv(filepath + '//' + fName, usecols=["Q_DHNf_W"]).values
```

Comment:
use `os.path.join` to create paths. did you run this? I don't think it **can** work - `'//'` is not a valid path separator. this is exactly why we use `os.path.join` - to make it portable (windows: `'\\'`, unix: `'/'`) 

also, while we're updating variable names... `fName` is *so* not pep8. how about `file_name`? or even better, use a name that specifies what kind of file we're talking about. `filepath`?  should be at least `file_path`, but also, *what* path???